code/datums/diseases/transformation.dm

```diff
@@ -189,7 +189,7 @@
 	stage2	= list("You are turning a little green.")
 	stage3	= list("<span class='danger'>Your limbs are getting oozy.</span>", "<span class='danger'>Your skin begins to peel away.</span>")
 	stage4	= list("<span class='danger'>You are turning into a slime.</span>")
-	stage5	= list("<span class='danger'>You have become a slime.</span>")
+	stage5	= list("<span class='danger'>You become a slime.</span>")

```

Comment:
the phrasing on these messages is a nethack reference

It's not a huge deal, I just wanted to point out that's why it's like that